test/presenters/html_publication_presenter_test.rb

```diff
@@ -39,4 +39,25 @@ def format_name
   test 'has organisation branding' do
     assert presented_item("published").is_a?(OrganisationBranding)
   end
+
+  test 'includes custom organisation logos when a single organisation is listed' do
+    presented = presented_item("updated")
+    organisation = presented.organisations.first
+    example_logo = schema_item("updated")["links"]["organisations"][0]["details"]["logo"]["image"].symbolize_keys
+    presented_logo = presented.organisation_logo(organisation)[:organisation][:image]
+
+    refute presented.organisations.count > 1
```

Comment:
This hurts my head a bit. `assert count < 2` or `assert_equal 1, presented.organisations.count`?